js/src/main/scala/Main.scala

```diff
@@ -144,7 +161,7 @@ object Main {
 
   def checkProgramType(pgrm: Pgrm): Ls[Option[Str] -> Str] -> Option[Str] = {
     val (diags, (typeDefs, stmts)) = pgrm.desugared
-    
```

Comment:
Is there a scalafmt option not to remove trailing indentation spaces? Removing them makes editing harder for those not using an automated formatter.

Reply:
I didn't find related options. Reverted.